### ISSUE TITLE
write tif

### ISSUE THREAD

**Issue by 464hee** [cgohlke/tifffile#174]
Can Tifffile write a tile to any location? What function should I use to specify the writing position

**Comment by cgohlke** [comment]
No. It is not possible to write a tile to any location/position with tifffile.

**Comment by cgohlke** [comment]
> we would prefer, to write only the valid tissue regions,

According to the TIFF standard, all tiles must be written in full size. The GeoTIFF standard supports a sparse format where tiles with a 0 offsets and bytecounts default to a "nodata" value. Tifffile supports reading an writing such files but AFAIK not many software outside the GIS domain is able to read such files.

**Comment by cowrturn** [comment]
That's great, can you put out a demo sometime?  As written in README.rst:

>>> series0 = numpy.random.randint(0, 255, (32, 32, 3), 'uint8')
>>> series1 = numpy.random.randint(0, 255, (4, 256, 256), 'uint16')
>>> with TiffWriter('temp.tif') as tif:
...     tif.write(series0, photometric='rgb')
...     tif.write(series1, photometric='minisblack')

**Comment by cgohlke** [comment]
Provide a generator of tiles where empty tiles are None. There's an example in the tests:

https://github.com/cgohlke/tifffile/blob/103b6a337db6a84293562bda65fdc8c382281fe2/tests/test_tifffile.py#L16147-L16179
